client/interactive.js

```diff
@@ -69,36 +67,34 @@ export class FormulaBunBotInteracive extends FormulaBunBotBase {
 
   set error(err) {
     if (!this.readyAt) return;
-    this.user.setActivity("for a heartbeat", { type: "LISTENING" }).then(() => {
-      this.user.setStatus("dnd");
-    });
+    this.user.setActivity('for a heartbeat', { type: 'LISTENING' });
+    this.user.setStatus('dnd');
   }
 
-  respond(interaction) {
-    const command = interaction.data.name;
+  async respond(interaction) {
+    const command = interaction.commandName;
 
     if (!this.server.serverinfo || !this.server.playerinfo) {
       return;
     }
 
     if (commands[command]) {
+      await interaction.deferReply();
+
       const response = commands[command].respond(this.server);
 
-      this.api.interactions(interaction.id, interaction.token).callback.post({
-        data: {
-          type: 4,
-          data: {
-            content: response.content,
-            embeds: response.embed,
-          },
-        },
+      await interaction.editReply({
```

Comment:
I love this api :smiling_face_with_tear:

Reply:
Did this to avoid timeouts from the bot being slow.